sql/core/src/test/scala/org/apache/spark/sql/execution/benchmark/AggregateBenchmark.scala

```diff
@@ -73,23 +73,26 @@ object AggregateBenchmark extends SqlBasedBenchmark {
         spark.range(N).selectExpr("(id & 65535) as k").groupBy("k").sum().collect()
       }
 
-      benchmark.addCase(s"codegen = F", numIters = 2) { iter =>
-        spark.conf.set("spark.sql.codegen.wholeStage", "false")
-        f()
+      benchmark.addCase(s"codegen = F", numIters = 2) { _ =>
+        withSQLConf(SQLConf.WHOLESTAGE_CODEGEN_ENABLED.key -> false.toString) {
```

Comment:
`"false"` instead of `false.toString`?

Reply:
When we use `Seq(true, false).foreach { value =>`, we usually do `s"$value"`. But, for this, I think `"false"` is the simplest and the best.